sources/INSPECT

```diff
@@ -111,22 +117,21 @@ Copyright (c) 1982-1987, 1990-1991, 1993, 1995, 1999, 2018, 2021 by Venue & Xero
                                                ((OR (NULL PROPERTIES)
                                                     (LISTP PROPERTIES))
                                                 PROPERTIES)
-                                               (T            (* ; 
-                                                           "allow PROPERTIES to be a function")
+                                               (T            (* ; "allow PROPERTIES to be a function")
                                                   (APPLY* PROPERTIES DATUM]
-                      (SETQ VALUEMARGIN (\INSPECTW.VALUE.MARGIN
-                                         (COND
-                                            (PROPPRINTFN (for PROP in PROPERTIESLST
-                                                            collect (APPLY* PROPPRINTFN PROP 
-                                                                               DATUM)))
-                                            (T PROPERTIESLST))
-                                         IWFONT))
+                      (SETQ VALUEMARGIN (\INSPECTW.VALUE.MARGIN (COND
+                                                                   (PROPPRINTFN
+                                                                    (for PROP in PROPERTIESLST
```

Comment:
it's distubing to see the pretty printer mangle things so badly that looked OK before.
No chance to review these using lisp's PRC, just saying